openrtb_ext/request_wrapper.go

```diff
@@ -780,7 +832,7 @@ func (re *RegExt) SetUSPrivacy(uSPrivacy string) {
 type SiteExt struct {
 	ext      map[string]json.RawMessage
 	extDirty bool
-	amp      int8
+	amp      *int8
```

Comment:
We spoke about this yesterday, but I just wanted to confirm: this was changed to a pointer so we could check if this `amp` variable is empty?